content/spin/v2/cli-reference.md

```diff
@@ -1850,4 +2583,22 @@ CLI commands have four phases that indicate levels of stability:
 
 {{ blockEnd }}
 
+{{ startTab "v2.3"}}
+
+| Command                                                                               | Stability    |
+| ------------------------------------------------------------------------------------- | ------------ |
+| <code>spin add</code>                                                                 | Stable       |
+| <code>spin build</code>                                                               | Stable       |
+| <code>spin new</code>                                                                 | Stable       |
+| <code>spin plugins</code>                                                             | Stable       |
+| <code>spin templates</code>                                                           | Stable       |
+| <code>spin up</code>                                                                  | Stable       |
+| <code>spin cloud</code>                                                               | Stabilizing  |
+| <code>spin registry</code>                                                            | Stabilizing  |
+| <code>spin doctor</code>                                                              | Experimental |
+| <code>spin watch</code>                                                               | Experimental |
+| <code>spin bindle</code>                                                              | Deprecated   |
```

Comment:
Hmm.  I'm pretty sure `spin bindle` hasn't existed since 2.0.

Reply:
Thanks, removed the Bindle reference.